dts/bindings/riscv/nordic,vpr-coprocessor.yaml

```
@@ -0,0 +1,32 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+compatible: "nordic,vpr-coprocessor"
+
+description: |
+  VPR coprocessor
+
+  VPR is a RISC-V CPU implementation. VPR instances are exposed to other CPUs as
+  peripherals. A domain can take ownership of one or more VPRs.
```

```suggestion
  VPR is a RISC-V CPU implementation.
```
Not true for all VPRs.

---

We should probably name this compatible `nordic,nrf-vpr-coprocessor`, as it's for the VPR seen as peripheral from nRF SoCs (not the core itself, which has its own binding).

---

My understanding is that this compatible is used to describe two things at the same time:
- how to configure security and ownership (nRF54H specific)
- how to run VPR (applies to both nRF54H and nRF54L)

Perhaps the proper way would be to split those into separate bindings?

---

> * how to configure security and ownership (nRF54H specific)

Not really. nrf-regtool doesn't need an extra binding for that specifically, when it comes to VPRs. The only thing specific to nRF54H is the `cpu` property, which is really just hardware description.

---

Ok, the issue with H-specific stuff is addressed.
But I think the description of node should be changed - at least last sentence has to be removed.

---

Removed the last sentence.